### PR TITLE
Python 3 compat rebase

### PR DESCRIPTION
**Pull request by benjwadams**
Updates Glider DAC Status application for Python 3 compatibility.
Also switches out Flask-Environments for Dynaconf due to incompatibilities with the former in newer versions of Flask.  Supercedes previous PR in #28

### CODE REVIEW

File: web/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,2494 @@
+{
```
Comment (benjwadams) [inline web/package-lock.json]: I didn't realize this file got included in the original PR in #28 for some reason.  We should likely remove this.

File: manage.py [hunk 2]
```diff
@@ -29,11 +29,11 @@ def list_routes():
 
         methods = ','.join(rule.methods)
         url = url_for(rule.endpoint, **options)
-        line = urllib.unquote("{:50s} {:20s} {}".format(rule.endpoint, methods, url))
+        line = urllib.parse.unquote("{:50s} {:20s} {}".format(rule.endpoint, methods, url))
         output.append(line)
 
     for line in sorted(output):
-        print line
+        print(line)
```
Comment (Bobfrat) [inline manage.py]: Do we really need this for loop?

**Comment by Bobfrat** [comment]
Looks good, can we remove the print statements all together? If its something worth logging, let's log it (and rotate the logs)

File: manage.py (hunk 2 above)
Comment (benjwadams) [reply manage.py]: Probably not, but the purpose of this commit is to upgrade to Python 3.  We can remove in a separate commit if need be.

**Pull request merged**